docs/2_QUICK_START_GUIDE.md

```diff
@@ -1,13 +1,229 @@
+
+---
+
+## 2.4 Deploy Cobalt's [_Azure Hello World CIT_](../infra/templates/az-hello-world/README.md)
+
+Below are the steps for deploying the [_Azure Hello World CIT_](../infra/templates/az-hello-world/README.md). Ensure that this deployment outputs the following Azure cloud resources before you can call this quick start guide finished:
+
+| Preview | Quick Start Azure Cloud Resources |
+|----------|--------------|
+|![cobalt-'NickeManarin/Screen2Gif'](./screentogif-40sec-test.gif)| <image src="https://user-images.githubusercontent.com/10041279/66533295-0fc64b80-ead8-11e9-96c0-ea5441cde854.png" width="1400"/> |
+
+<!--- These gifs will have screenshots from forking, terraform plan and apply, visit azure portal and visit app service url --->
+
+### **Step 1:** Fork Cobalt Repo
+
+Initializing a repository that you own and control is recommended. Once forked, move onto the next step.
+
+* From any page within this repository, find the forked menu and fork the repo into your own repository.
+
+    ![image](https://user-images.githubusercontent.com/10041279/66366857-6e17f080-e957-11e9-8b32-266b0d4a98fc.png)
+
+### **Step 2:** Clone Repo to Local Environment
+
+You will be deploying Azure infrastructure for your local environment so you will need to have a copy of the Cobalt project locally.
+
+* From any terminal with git, use the following git command to clone your forked repo into your local environment. You can find your git repo url at the landing page of your forked repo.
+
+    ```bash
+    git clone <insert-forked-git-repo-url> # ex. git clone https://github.com/<YourGitAccout>/cobalt.git
+    ```
+
+### **Step 3:** Setup Local Environment Variables
+
+You'll need to define a `.env` file in the root of your local project. You can use our [environment template file](https://github.com/microsoft/cobalt/blob/master/.env.template) to start. This will hold all the environment variables needed to run your Cobalt deployments locally.
+
+1. Navigate to the root directory of your project and use the following command to copy the environment template file.
+
+    ```bash
+    cp .env.template .env
+    ```
+
+    ```bash
+    $ tree infra
+    ├───.env.template
+    ├───.env ## NEW FILE GENERATED FROM THE COMMAND
+    ├───modules
+    │   └───providers
+    │       ├───azure
+    │       │   ├───...
+    │       │   └───vnet
+    │       └───common
+    └───templates
+        ├───az-hello-world
+        │   └───test
+        └───backend-state-setup
+    ```
+
+1. Provide values for the environment values in `.env` which are required to authenticate Terraform to provision resources within your subscription.
+
+    ```bash
```

Comment:
Instead of dumping it here, it may be useful to comment what these are in the `.env.template` and just add a step to fill out the values called out in the template. 

That way as the template changes, the docs don't need to be updated